src/linalg.jl

```diff
@@ -76,12 +76,21 @@ function LinearAlgebra.rdiv!(
     A::Matrix{T},
     adjB::Adjoint{T,<:LowerTriangular{T,UniformBlockDiagonal{T}}},
 ) where {T}
+    m, n = size(A)
     Bd = adjB.parent.data
-    m, n, k = size(Bd.data)
-    size(A, 2) == size(Bd, 1) && m == n || throw(DimensionMismatch(""))
-    inds = 1:m
-    for (i, f) in enumerate(Bd.facevec)
-        BLAS.trsm!('R', 'L', 'T', 'N', one(T), f, view(A, :, inds .+ m * (i - 1)))
+    r, s, blk = size(Bd.data)
+    n == size(Bd, 1) && r == s || throw(DimensionMismatch())
+    @inbounds for b = 1:blk
```

Comment:
I like the comment below on which multiplication these loops are replacing. Can you do the same here?